README.md

```diff
@@ -621,7 +619,10 @@ echo "Operation status: ".$status->getStatus()."<br />";
 ```
 
 ##Media Services
-
+> **Note**
+> 
+>The current Media Services API only works with PHP 5.5 and above.
```

Comment:
That does make sense. The default PHP version is 5.4 on Azure and only Media Services API does not support it, also the Media Services API is not very widely used. That is why we want to keep the package at 5.4. I am planning to fix the code in Media Services so that it will support 5.4. As far as I know, the Media Services code uses empty() function that is not compatible with v5.4. It would not be too bad to fix it.